lib/models/issueish.js

```diff
@@ -0,0 +1,65 @@
+import {URL} from 'url';
+import moment from 'moment';
+
+import {category} from '../containers/pr-statuses-container';
+
+export default class Issueish {
+  constructor(data) {
+    this.number = data.number;
+    this.title = data.title;
+    this.url = new URL(data.url);
+    this.authorLogin = data.author.login;
+    this.authorAvatarURL = new URL(data.author.avatarUrl);
+    this.createdAt = moment(data.createdAt, moment.ISO_8601);
+    this.headRefName = data.headRefName;
+    this.headRepositoryID = data.repository.id;
+    this.statusContexts = data.commits.nodes.reduce((acc, node) => {
+      const status = node.commit.status;
+      if (status !== null) {
+        acc.push(...status.contexts);
+      }
+      return acc;
+    }, []);
+  }
+
+  getNumber() {
+    return this.number;
+  }
+
+  getTitle() {
+    return this.title;
+  }
+
+  getGitHubURL() {
+    return this.url.toString();
+  }
+
+  getAuthorLogin() {
+    return this.authorLogin;
+  }
+
+  getAuthorAvatarURL(size = 32) {
```

Comment:
I wonder if we are getting the author avatar URL in a bunch of places in .our codebase.  if so, might not hurt to DRY it up. (Not something that needs to be addressed as part of this PR, just thinking out loud about future cleanup.)